elements/bulbs-video/elements/video-carousel.js

```diff
@@ -68,8 +71,9 @@ class BulbsVideoCarousel extends BulbsHTMLElement {
 
   firstPlay () {
     let items = this.querySelectorAll('bulbs-carousel-item');
+    let nowPlayingItems = this.querySelectorAll('[now-playing]');
 
-    if (items.length > 0) {
+    if (nowPlayingItems.length > 0 && items.length > 0) {
```

Comment:
Which calls through to this. The apply state function is doing two things:

1: updating the styles on the carousel item to highlight the current playing item.
2: sets `src` attribute of the video player. Changing the `src` attribute of a `<bulbs-video>` tears down the jwplayer instance and creates a new one.